src/web/routes/admin/teams/teams-router.ts

```diff
@@ -45,7 +40,6 @@ teamsRouter.post(
 		res: ConnectFigmaTeamResponse,
 		next: NextFunction,
 	) => {
-		assertSchema(req.params, CONNECT_FIGMA_TEAM_ROUTE_PARAMS_SCHEMA);
```

Comment:
**note:** Remove schema validation for URL parameters as it is handled by Express.